src/main/java/controller/RateList.java

```diff
@@ -0,0 +1,98 @@
+package controller;
+
+import model.*;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Created by aleksey.dobrovolsky on 6/9/2019.
+ */
+public class RateList {
+
+    private static ArrayList<Rate> rateList = createRateList();
+    private static DecimalFormat df2 = new DecimalFormat("#.##");
+    private static Properties p = new Properties();
+
+    private static FileReader readProperties() throws FileNotFoundException {
+        return new FileReader(System.getProperty("user.dir") + "/src/main/resources/rateFilterData");
+    }
+
+    public static void run() throws IOException {
+        System.out.println("Initial list of rates:");
+        controller.RateList.printList(rateList);
```

Comment:
why don't you just use printList(..) instead of controller.RateList.printList(..). It's overhead.

Reply:
done - printList is called as a method of StudetsList objects without any attributes